### ISSUE TITLE
Xiaomi Vibration sensor expose only programmable switch (and not motion sensor)

### ISSUE THREAD

**Issue by orevron** [itavero/homebridge-z2m#381]
**Describe the bug**
I added new Xiaomi Vibrations sensor (model [DJT11LM](https://www.zigbee2mqtt.io/devices/DJT11LM.html#xiaomi-djt11lm)) and the only thing I see on HomeKit is a programmable switch with 3 actions for (drop, tilt, and vibration).
Although it states in the [docs](https://github.com/itavero/homebridge-z2m/blob/master/docs/devices/xiaomi/djt11lm.md) for this device that a MotionSensor should be exposed too

**Related devices**
Aqara vibration sensor [DJT11LM](https://www.zigbee2mqtt.io/devices/DJT11LM.html#xiaomi-djt11lm)

**To Reproduce**
Add this device to z2m, and see the added device in the HomeKit app.

**Expected behavior**
Motion Sensor should be exposed too

**MQTT messages**
* The device entry from `zigbee2mqtt/bridge/devices`
```json
[
  {
    "definition": {
      "description": "Aqara vibration sensor",
      "exposes": [
        {
          "access": 1,
          "description": "Remaining battery in %",
          "name": "battery",
          "property": "battery",
          "type": "numeric",
          "unit": "%",
          "value_max": 100,
          "value_min": 0
        },
        {
          "access": 1,
          "description": "Triggered action (e.g. a button click)",
          "name": "action",
          "property": "action",
          "type": "enum",
          "values": [
            "vibration",
            "tilt",
            "drop"
          ]
        },
        {
          "access": 1,
          "name": "strength",
          "property": "strength",
          "type": "numeric"
        },
        {
          "access": 3,
          "name": "sensitivity",
          "property": "sensitivity",
          "type": "enum",
          "values": [
            "low",
            "medium",
            "high"
          ]
        },
        {
          "access": 1,
          "name": "angle_x",
          "property": "angle_x",
          "type": "numeric",
          "value_max": 90,
          "value_min": -90
        },
        {
          "access": 1,
          "name": "angle_y",
          "property": "angle_y",
          "type": "numeric",
          "value_max": 90,
          "value_min": -90
        },
        {
          "access": 1,
          "name": "angle_z",
          "property": "angle_z",
          "type": "numeric",
          "value_max": 90,
          "value_min": -90
        },
        {
          "access": 1,
          "description": "Voltage of the battery in millivolts",
          "name": "voltage",
          "property": "voltage",
          "type": "numeric",
          "unit": "mV"
        },
        {
          "access": 1,
          "description": "Link quality (signal strength)",
          "name": "linkquality",
          "property": "linkquality",
          "type": "numeric",
          "unit": "lqi",
          "value_max": 255,
          "value_min": 0
        }
      ],
      "model": "DJT11LM",
      "options": [],
      "supports_ota": false,
      "vendor": "Xiaomi"
    },
    "endpoints": {
      "1": {
        "bindings": [],
        "clusters": {
          "input": [],
          "output": []
        },
        "configured_reportings": [],
        "scenes": []
      }
    },
    "friendly_name": "office_vibration",
    "ieee_address": "0x00158d0006370b01",
    "interview_completed": true,
    "interviewing": false,
    "manufacturer": "LUMI",
    "model_id": "lumi.vibration.aq1",
    "network_address": 23546,
    "power_source": "Battery",
    "supported": true,
    "type": "EndDevice"
  }
]
```
* Status update from `zigbee2mqtt/[FRIENDLY_NAME]`
```
debug 2022-02-04 22:40:35: Received Zigbee message from 'office_vibration', type 'attributeReport', cluster 'closuresDoorLock', data '{"1285":1769472}' from endpoint 1 with groupID 0
info  2022-02-04 22:40:35: MQTT publish: topic 'home/office_vibration', payload '{"angle":8,"angle_x":2,"angle_x_absolute":88,"angle_y":2,"angle_y_absolute":88,"angle_z":87,"battery":80,"linkquality":90,"strength":27,"voltage":2965}'
```

**Versions used**
Please provide the version of the following pieces of software:
 - This plugin 1.6.2
 - Homebridge 1.4.0
 - Zigbee2MQTT 1.22.0
 - Homebridge Config UI X 4.42.0

**Comment by itavero** [comment]
I have this sensor myself and for me it does show up as a motion sensor as well as a Stateless Programmable Switch. Note that motion sensors don't show up as tiles, but are grouped above the tiles:
![image](https://user-images.githubusercontent.com/203398/152601451-10eb3ac8-3dfc-4b0b-b964-f1061d51e35a.jpeg)

As you can see here, mine has a "Bewegingssensor" (Dutch for motion sensor):
![image](https://user-images.githubusercontent.com/203398/152601565-7ae144f8-4d0c-432f-80c6-b7c243fcfec9.jpeg)

If you still can't find it, please turn in debug logging in Homebridge and share the lights if Homebridge starting up (when the accessories are restored/created).
This might provide some insight in what's going on on your end.

**Comment by orevron** [comment]
Thanks for answering! 
This is the homebridge debug logs 
```
[04/02/2022, 23:00:17] Registering platform 'homebridge-z2m.zigbee2mqtt'
[04/02/2022, 23:00:17] [zigbee2mqtt] Initializing zigbee2mqtt platform...
[04/02/2022, 23:00:17] [zigbee2mqtt] Default device config: {"excluded":false,"exclude":false}
[04/02/2022, 23:00:17] [zigbee2mqtt] Connecting to MQTT server at mqtt://192.168.1.190:1883
[04/02/2022, 23:00:17] [zigbee2mqtt] Using MQTT keepalive: 60
[04/02/2022, 23:00:17] [zigbee2mqtt] MQTT reject_unauthorized set false, ignoring certificate warnings.
[04/02/2022, 23:00:18] [zigbee2mqtt] Restoring accessory: office_button (0x00158d0002c48270)
[04/02/2022, 23:00:18] [zigbee2mqtt] Config for accessory office_button : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:18] [zigbee2mqtt] Mapping of property 'action' of device 'office_button':
Button 1:
- SINGLE: single
- DOUBLE: double
- LONG  : hold
Button 2 #ext1:
- SINGLE: triple
- DOUBLE: quadruple
Button 3 (many):
- SINGLE: many
[04/02/2022, 23:00:18] [zigbee2mqtt] Configuring Battery Service for office_button
[04/02/2022, 23:00:18] [zigbee2mqtt] Updating service names for office_button (from office_button)
[04/02/2022, 23:00:18] [zigbee2mqtt] Restoring accessory: office_door (0x00158d000588ad0d)
[04/02/2022, 23:00:18] [zigbee2mqtt] Config for accessory office_door : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:18] [zigbee2mqtt] Configuring ContactSensor for office_door
[04/02/2022, 23:00:18] [zigbee2mqtt] Configuring Battery Service for office_door
[04/02/2022, 23:00:18] [zigbee2mqtt] Updating service names for office_door (from office_door)
[04/02/2022, 23:00:18] [zigbee2mqtt] Restoring accessory: network_temp (0x00158d00042d5543)
[04/02/2022, 23:00:18] [zigbee2mqtt] Config for accessory network_temp : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:18] [zigbee2mqtt] Configuring HumiditySensor for network_temp
[04/02/2022, 23:00:18] [zigbee2mqtt] Configuring TemperatureSensor for network_temp
[04/02/2022, 23:00:18] [zigbee2mqtt] Configuring AirPressureSensor for network_temp
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring Battery Service for network_temp
[04/02/2022, 23:00:19] [zigbee2mqtt] Updating service names for network_temp (from network_temp)
[04/02/2022, 23:00:19] [zigbee2mqtt] Restoring accessory: office_vibration (0x00158d0006370b01)
[04/02/2022, 23:00:19] [zigbee2mqtt] Config for accessory office_vibration : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:19] [zigbee2mqtt] Mapping of property 'action' of device 'office_vibration':
Button 1 (drop):
- SINGLE: drop
Button 2 (tilt):
- SINGLE: tilt
Button 3 (vibration):
- SINGLE: vibration
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring Battery Service for office_vibration
[04/02/2022, 23:00:19] [zigbee2mqtt] Updating service names for office_vibration (from office_vibration)
[04/02/2022, 23:00:19] [zigbee2mqtt] Restoring accessory: office_motion (0x00158d00027ce768)
[04/02/2022, 23:00:19] [zigbee2mqtt] Config for accessory office_motion : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring OccupancySensor for office_motion
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring Battery Service for office_motion
[04/02/2022, 23:00:19] [zigbee2mqtt] Updating service names for office_motion (from office_motion)
[04/02/2022, 23:00:19] [zigbee2mqtt] Restoring accessory: entrance_switch (0x00158d0005c10071)
[04/02/2022, 23:00:19] [zigbee2mqtt] Config for accessory entrance_switch : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:19] [zigbee2mqtt] Mapping of property 'action' of device 'entrance_switch':
Button 1 (both):
- SINGLE: single_both
- DOUBLE: double_both
- LONG  : hold_both
Button 2 (left):
- SINGLE: single_left
- DOUBLE: double_left
- LONG  : hold_left
Button 3 (right):
- SINGLE: single_right
- DOUBLE: double_right
- LONG  : hold_right
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring Battery Service for entrance_switch
[04/02/2022, 23:00:19] [zigbee2mqtt] Updating service names for entrance_switch (from entrance_switch)
[04/02/2022, 23:00:19] [zigbee2mqtt] Restoring accessory: tom_button (0x00158d00035563b1)
[04/02/2022, 23:00:19] [zigbee2mqtt] Config for accessory toms_button : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:19] [zigbee2mqtt] Mapping of property 'action' of device 'toms_button':
Button 1:
- SINGLE: single
- DOUBLE: double
- LONG  : hold
Button 2 #ext1:
- SINGLE: triple
- DOUBLE: quadruple
Button 3 (many):
- SINGLE: many
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring Battery Service for toms_button
[04/02/2022, 23:00:19] [zigbee2mqtt] Updating service names for toms_button (from toms_button)
[04/02/2022, 23:00:19] [zigbee2mqtt] Restoring accessory: office_temp (0x00158d0005828de5)
[04/02/2022, 23:00:19] [zigbee2mqtt] Config for accessory office_temp : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring HumiditySensor for office_temp
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring TemperatureSensor for office_temp
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring AirPressureSensor for office_temp
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring Battery Service for office_temp
[04/02/2022, 23:00:19] [zigbee2mqtt] Updating service names for office_temp (from office_temp)
[04/02/2022, 23:00:19] [zigbee2mqtt] Restoring accessory: bedroom_temp (0x00158d0001d6510a)
[04/02/2022, 23:00:19] [zigbee2mqtt] Config for accessory bedroom_temp : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring HumiditySensor for bedroom_temp
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring TemperatureSensor for bedroom_temp
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring AirPressureSensor for bedroom_temp
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring Battery Service for bedroom_temp
[04/02/2022, 23:00:19] [zigbee2mqtt] Updating service names for bedroom_temp (from bedroom_temp)
[04/02/2022, 23:00:19] [zigbee2mqtt] Restoring accessory: 0x00158d00019d4aa8 (0x00158d00019d4aa8)
[04/02/2022, 23:00:19] [zigbee2mqtt] Config for accessory livingroom_temp : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring HumiditySensor for livingroom_temp
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring TemperatureSensor for livingroom_temp
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring AirPressureSensor for livingroom_temp
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring Battery Service for livingroom_temp
[04/02/2022, 23:00:19] [zigbee2mqtt] Updating service names for livingroom_temp (from livingroom_temp)
[04/02/2022, 23:00:19] [zigbee2mqtt] Restoring accessory: 0x00158d00024b0cb1 (0x00158d00024b0cb1)
[04/02/2022, 23:00:19] [zigbee2mqtt] Config for accessory toms_door : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring ContactSensor for toms_door
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring Battery Service for toms_door
[04/02/2022, 23:00:19] [zigbee2mqtt] Updating service names for toms_door (from toms_door)
[04/02/2022, 23:00:19] [zigbee2mqtt] Restoring accessory: 0x00158d0002c4995d (0x00158d0002c4995d)
[04/02/2022, 23:00:19] [zigbee2mqtt] Config for accessory entrance_door : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring ContactSensor for entrance_door
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring Battery Service for entrance_door
[04/02/2022, 23:00:19] [zigbee2mqtt] Updating service names for entrance_door (from entrance_door)
[04/02/2022, 23:00:19] [zigbee2mqtt] Restoring accessory: 0x00158d00035afe4c (0x00158d00035afe4c)
[04/02/2022, 23:00:19] [zigbee2mqtt] Config for accessory kitchen_window : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring ContactSensor for kitchen_window
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring Battery Service for kitchen_window
[04/02/2022, 23:00:19] [zigbee2mqtt] Updating service names for kitchen_window (from kitchen_window)
[04/02/2022, 23:00:19] [zigbee2mqtt] Restoring accessory: livingroom_window (0x00158d00035a4e55)
[04/02/2022, 23:00:19] [zigbee2mqtt] Config for accessory livingroom_window : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring ContactSensor for livingroom_window
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring Battery Service for livingroom_window
[04/02/2022, 23:00:19] [zigbee2mqtt] Updating service names for livingroom_window (from livingroom_window)
[04/02/2022, 23:00:19] [zigbee2mqtt] Restoring accessory: 0x00158d00035a4e38 (0x00158d00035a4e38)
[04/02/2022, 23:00:19] [zigbee2mqtt] Config for accessory bedroom_window : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring ContactSensor for bedroom_window
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring Battery Service for bedroom_window
[04/02/2022, 23:00:19] [zigbee2mqtt] Updating service names for bedroom_window (from bedroom_window)
[04/02/2022, 23:00:19] [zigbee2mqtt] Restoring accessory: 0x00158d0002c498d4 (0x00158d0002c498d4)
[04/02/2022, 23:00:19] [zigbee2mqtt] Config for accessory livingroom_balcony_right : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring ContactSensor for livingroom_balcony_right
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring Battery Service for livingroom_balcony_right
[04/02/2022, 23:00:19] [zigbee2mqtt] Updating service names for livingroom_balcony_right (from livingroom_balcony_right)
[04/02/2022, 23:00:19] [zigbee2mqtt] Restoring accessory: 0x00158d00034d84cc (0x00158d00034d84cc)
[04/02/2022, 23:00:19] [zigbee2mqtt] Config for accessory livingroom_balcony_left : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring ContactSensor for livingroom_balcony_left
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring Battery Service for livingroom_balcony_left
[04/02/2022, 23:00:19] [zigbee2mqtt] Updating service names for livingroom_balcony_left (from livingroom_balcony_left)
[04/02/2022, 23:00:19] [zigbee2mqtt] Restoring accessory: 0x00158d000588c7fe (0x00158d000588c7fe)
[04/02/2022, 23:00:19] [zigbee2mqtt] Config for accessory bedroom_door : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring ContactSensor for bedroom_door
[04/02/2022, 23:00:19] [zigbee2mqtt] Configuring Battery Service for bedroom_door
[04/02/2022, 23:00:19] [zigbee2mqtt] Updating service names for bedroom_door (from bedroom_door)
[04/02/2022, 23:00:20] [zigbee2mqtt] Restoring accessory: 0x00158d0005890d1b (0x00158d0005890d1b)
[04/02/2022, 23:00:20] [zigbee2mqtt] Config for accessory bedroom_motion : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:20] [zigbee2mqtt] Configuring OccupancySensor for bedroom_motion
[04/02/2022, 23:00:20] [zigbee2mqtt] Configuring Battery Service for bedroom_motion
[04/02/2022, 23:00:20] [zigbee2mqtt] Updating service names for bedroom_motion (from bedroom_motion)
[04/02/2022, 23:00:20] [zigbee2mqtt] Restoring accessory: 0x00158d0005890e16 (0x00158d0005890e16)
[04/02/2022, 23:00:20] [zigbee2mqtt] Config for accessory toms_motion : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:20] [zigbee2mqtt] Configuring OccupancySensor for toms_motion
[04/02/2022, 23:00:20] [zigbee2mqtt] Configuring Battery Service for toms_motion
[04/02/2022, 23:00:20] [zigbee2mqtt] Updating service names for toms_motion (from toms_motion)
[04/02/2022, 23:00:20] [zigbee2mqtt] Restoring accessory: bedroom_button (0x00158d0003fb70a2)
[04/02/2022, 23:00:20] [zigbee2mqtt] Config for accessory bedroom_button : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:20] [zigbee2mqtt] Mapping of property 'action' of device 'bedroom_button':
Button 1:
- SINGLE: single
- DOUBLE: double
- LONG  : hold
Button 2 #ext1:
- SINGLE: triple
- DOUBLE: quadruple
Button 3 (many):
- SINGLE: many
[04/02/2022, 23:00:20] [zigbee2mqtt] Configuring Battery Service for bedroom_button
[04/02/2022, 23:00:20] [zigbee2mqtt] Updating service names for bedroom_button (from bedroom_button)
[04/02/2022, 23:00:20] [zigbee2mqtt] Restoring accessory: livingroom_motion (0x00158d00027d1213)
[04/02/2022, 23:00:20] [zigbee2mqtt] Config for accessory livingroom_motion : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:20] [zigbee2mqtt] Configuring OccupancySensor for livingroom_motion
[04/02/2022, 23:00:20] [zigbee2mqtt] Configuring Battery Service for livingroom_motion
[04/02/2022, 23:00:20] [zigbee2mqtt] Updating service names for livingroom_motion (from livingroom_motion)
[04/02/2022, 23:00:20] [zigbee2mqtt] Restoring accessory: office_window (0x00158d0002c10b55)
[04/02/2022, 23:00:20] [zigbee2mqtt] Config for accessory office_window : {"excluded":false,"exclude":false}
[04/02/2022, 23:00:20] [zigbee2mqtt] Configuring ContactSensor for office_window
[04/02/2022, 23:00:20] [zigbee2mqtt] Configuring Battery Service for office_window
[04/02/2022, 23:00:20] [zigbee2mqtt] Updating service names for office_window (from office_window)
[04/02/2022, 23:00:20] [zigbee2mqtt] Connected to MQTT server
[04/02/2022, 23:00:20] [zigbee2mqtt] Using Zigbee2MQTT v1.22.0 (identified via home/bridge/info)
[04/02/2022, 23:00:20] [zigbee2mqtt] Zigbee2MQTT 'experimental.output' is set to 'json'
[04/02/2022, 23:00:21] [zigbee2mqtt] Received devices...
[04/02/2022, 23:00:21] [zigbee2mqtt] Device is excluded: 0x00124b0024c9ea08
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: office_door/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: network_temperature/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: office vibration/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: office_vibration/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: office_motion/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: entrance_switch/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: tom_switch/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: office_button/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: tom_button/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: office_temp/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: network_temp/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: bedroom_temp/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: livingroom_temp/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: toms_door/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: entrance_door/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: kitchen_window/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: livingroom_window/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: bedroom_window/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: livingroom_balcony_right/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: livingroom_balcony_left/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: bedroom_door/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: bedroom_motion/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: toms_motion/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: toms_button/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: bedroom_button/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: livingroom_motion/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: office_window/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: zigbee router/availability
[04/02/2022, 23:00:22] [zigbee2mqtt] Unhandled message on topic: zigbee_router/availability
```

**Comment by orevron** [comment]
From [this](https://www.zigbee2mqtt.io/devices/DJT11LM.html#vibration-binary) I understand that the zigbee2mqtt should have vibration key in the published message, but the message in the mqtt server logs is missing this key.. 
```
info  2022-02-04 22:45:34: MQTT publish: topic 'home/office_vibration', payload '{"angle":8,"angle_x":2,"angle_x_absolute":88,"angle_y":2,"angle_y_absolute":88,"angle_z":87,"battery":80,"linkquality":87,"strength":19,"voltage":2965}'
```
you think it might be related? did you add any specific configuration in the zigbee2mqtt `configration.yaml` file?

**Comment by itavero** [comment]
Either it's not creating the motion sensor service on your environment or I forgot to add logging for it.
I did notice I'm running a slightly newer version of Zigbee2MQTT (v1.22.2) and this plugin (v1.7.0-rc.1).

As far as I know/can find, I haven't added any specific configuration in Zigbee2MQTT for this device.

**Comment by itavero** [comment]
Just noticed that your device's _exposes_ information doesn't include the `vibration` either. Perhaps updating Zigbee2MQTT can help.

**Comment by orevron** [comment]
First thing first :) happy B-day! 
I'll try what you suggested and dig more 
Thank you so much for trying !

**Comment by itavero** [comment]
I've checked the change log of Zigbee2MQTT and this was indeed fixed/added in v1.22.1.
Updating it should resolve your issue. Good luck!

**Comment by orevron** [comment]
Just saw that too in the changelog! thanks!
